docs/eventing/brokers/broker-types/kafka-broker/configuring-kafka-features.md

```diff
@@ -0,0 +1,206 @@
+# Configuring Kafka Features
+
+There are many different configuration options for how Knative Eventing and the Kafka Broker interact with Apache Kafka.
+
+## Configure Knative Eventing Kafka features
+
+There are various kafka features/default values the Knative Kafka Broker uses when interacting with Kafka.
+
+### Dispatcher rate limiter
+
+By default, there will be no rate limiting applied as events are dispatched from the data-plane. However, if you would like to enable rate limiting based on the number of
+virtual replicas, you can set `dispatcher.rate-limiter: "enabled"` in the config map below.
+
+### Dispatcher ordered execution metrics
+
+By default, only a subset of availabe metrics are collected from the data-plane event dispatcher. If you would like to enable additional metrics to track the ordered execution,
+you can set `dispatcher.ordered-executor-metrics: "enabled"` in the config map below.
+
+### Controller autoscaler
+
+By default, the controller will not autoscale the resources being used by the Kafka consumers within the data-plane. If you would like to enable autoscaling of these
+resources with [KEDA](https://keda.sh/), then you can set `controller.autoscaler: "enabled"` in the config map below.
```

Comment:
These 3 features needs more explanation, I don't think that as a user I'd understand, so I'd remove them for now (they are not completely implemented as well)